### PR TITLE
Fixed: Add a news section

### LINKED ISSUE #497: Create News Section on ocfl.io
We need a news section on the ocfl.io site for press releases. text for the press release can be found here: https://docs.google.com/document/d/1pNteQqcBkrkNedu-AVLy0izSpWJiJFxtq96Wa9UHz6Q/edit#

### PR DESCRIPTION
**Pull request by ahankinson**
Very basic news section with a copy of the press release

Fixes #497

### CODE REVIEW

File: news/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# News
+
+## Version 1.0 of the Oxford Common File Layout (OCFL) Released
```
Comment (zimeon) [inline news/index.md:3]: Needs date

**Comment by zimeon** [comment]
Looks great except for date

File: news/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+which provides a straightforward but comprehensive register for the object and versions of its content, and an OCFL 
+Storage Root, an arrangement for how OCFL Objects are laid out on physical storage. It also contains examples 
+illustrating the use of the OCFL, and explanations that ground decisions in prior experience. 
+
+The companion [OCFL Implementation Notes](https://ocfl.io/1.0/implementation-notes/) contains advice for implementing 
+the specification including recommendations for digital preservation, storage handling, client behaviors, and best 
+practices for dealing with OCFL Objects in motion.
+
+The OCFL Editors are also releasing [validation rules](https://ocfl.io/validation/validation-codes.html) and [fixture 
+objects](https://github.com/OCFL/fixtures) for testing OCFL implementations against the specification. We welcome your
```
Comment (rosy1280) [inline news/index.md:46]: I don't think this link will work over a line break?

**Comment by awoods** [comment]
It seems like we should be linking to this new `news/index.md` file from the main `index.md`, no?

File: news/index.md (hunk 1 above)
Comment (ahankinson) [reply news/index.md:3]: Fixed.

File: news/index.md (hunk 2 above)
Comment (ahankinson) [reply news/index.md]: Fixed

**Comment by ahankinson** [comment]
Added a link to the front page.

**Pull request merged**